lib/src/Uncertainty/Distribution/FisherSnedecor.cxx

```diff
@@ -195,6 +195,12 @@ void FisherSnedecor::computeMean() const
   isAlreadyComputedMean_ = true;
 }
 
+/* Compute the entropy of the distribution */
+Scalar FisherSnedecor::computeEntropy() const
+{
+  return std::log(d2_ / d1_) + SpecFunc::LogBeta(0.5 * d1_, 0.5 * d2_) + (1.0 - 0.5 * d1_) * SpecFunc::Psi(0.5 * d1_) + (0.5 * (d1_ + d2_)) * SpecFunc::Psi(0.5 * (d1_ + d2_)) - (1.0 + 0.5 * d2_) * SpecFunc::Psi(0.5 * d2_);
```

Comment:
Shouldn't it be
```
std::log(d1_  / d2_)
```
instead?

Reply:
You did not add the numerical test using computeLogPDF for every distributions, I guess that's why these were not checked.

Reply:
@jschueller I agree that some methods are not fully tested, but IMO it should go into another PR: this one is large enough and focused on computeEntropy(). BTW, it could be a shared work...

Reply:
I'm only referring to the test of computeEntropy:
```
fullprint << "entropy=" << distribution.computeEntropy() << std::endl;
fullprint << "entropy (MC)=" << -distribution.computeLogPDF(distribution.getSample(1000000)).computeMean()[0] << std::endl;
```

Reply:
@dbarbier There is an error... but in the formula given by wikipedia! One more... If you want a reliable source to check the formulas, you could have a look here:
[https://books.google.fr/books?id=k2fOBQAAQBAJ](url)
and if you look here:
[https://books.google.fr/books?id=k2fOBQAAQBAJ&printsec=frontcover&hl=fr&source=gbs_atb#v=onepage&q=fisher&f=false](url)
you get the formula in the code.

Reply:
@dbarbier note that I really appreciate your careful reading. Even if I did my best to check things, mistakes are likely to happen in so many formulas.

Reply:
@jschueller Ok I got it. I will add this test in all the C++ tests. I skipped this check in the Python tests in order to reduce the testing time. If you think I should add these checks here too just let me know.

Reply:
No, C++ is enough to check computations, thanks.
BTW in the book about Fisher distribution, there is a log2 factor; we really need some automated tests to have more confidence in these formulas.

Reply:
@dbarbier It is because the entropy can be computed according to any positive scale a, the base of the logarithm used in the definition of the entropy. The two most common choices are a=2, mainly in information theory (the entropy is given in bits), and a=e in probabilities (the entropy is given in nats, ie NATural logarithm).